browser/app/js/browser/__tests__/AboutModal.test.js

```diff
@@ -26,13 +26,13 @@ describe("AboutModal", () => {
   }
 
   it("should render without crashing", () => {
-    shallow(<AboutModal serverInfo={serverInfo} />)
+    shallow(<AboutModal serverInfo={serverInfo} t={key => key} />)
```

Comment:
This makes it to the component since without it, there is a `TypeError: t is not a function` error. This being said, it does not actually do any translation. Some tests are failing because it expects ""Link copied to clipboard!" but has 'msgCopied' (aka the key)